Update qTox Windows version building manual

But CMake adding exists in **Setting up PATH**.
https://github.com/tux3/qTox/blob/master/INSTALL.md#setting-up-path

I give up for now. I've got fully generated dependencies from manual, but I can't compile qTox. `qmake` behavior is strange, `make -j4` fails with error: 

> src/persistence/db/rawdatabase.cpp:15:31: fatal error: sqlcipher/sqlite3.h: No such file or directory

![2016-06-10_19-32-06](https://cloud.githubusercontent.com/assets/12829428/15965874/312d6498-2f42-11e6-8a32-e05463493b4b.png)

SQLCipher is not exist in MSYS2 repos.
**How do you building qTox for Windows from [build.tox.chat](https://build.tox.chat)?**

I am too struggling to build on Windows. After some addition to INSTALL.md, still something is missing.

Here's bootstrap.bat output: https://gist.github.com/Talkless/a78b687d9492fe99be39541ff46a31a3

Some interesting parts:

```
/cygdrive/c/MinGW/bin/autoheader: /cygdrive/c/MinGW/bin/autoheader-2.68: /bin/perl: bad interpreter: Permission denied
```

at line [162](https://gist.github.com/Talkless/a78b687d9492fe99be39541ff46a31a3#file-bootstrap_bat_output-txt-L162)

Also this compilation error:

```
C:/myqTox/libs/openal-soft-1.16.0/include/threads.h:38:8: error: redefinition of 'struct timespec'
```

at line [528](https://gist.github.com/Talkless/a78b687d9492fe99be39541ff46a31a3#file-bootstrap_bat_output-txt-L528)

More, not sure if critical:

```
/usr/bin/sed: -e expression #1, char 6: unterminated `s' command
```

at line [5842](https://gist.github.com/Talkless/a78b687d9492fe99be39541ff46a31a3#file-bootstrap_bat_output-txt-L5842)

More:

```
/usr/bin/mktemp: cannot create temp file /cygdrive/c/Users/IEUser/AppData/Local/Temp/ffconf.IqbHrgeU: No such file or directory
```

at line [5866](https://gist.github.com/Talkless/a78b687d9492fe99be39541ff46a31a3#file-bootstrap_bat_output-txt-L5866)

And finally, at the end of file:

```
Makefile:2: config.mak: No such file or directory
Makefile:59: /common.mak: No such file or directory
Makefile:100: /libavutil/Makefile: No such file or directory
Makefile:100: /library.mak: No such file or directory
Makefile:102: /doc/Makefile: No such file or directory
Makefile:185: /tests/Makefile: No such file or directory
make: *** No rule to make target `/tests/Makefile'.  Stop.
Makefile:2: config.mak: No such file or directory
Makefile:59: /common.mak: No such file or directory
Makefile:100: /libavutil/Makefile: No such file or directory
Makefile:100: /library.mak: No such file or directory
Makefile:102: /doc/Makefile: No such file or directory
Makefile:185: /tests/Makefile: No such file or directory
make: *** No rule to make target `/tests/Makefile'. Stop.
```

So.. basically nothing gets build during bootstrap.

--- a/windows/bootstrap.sh
+++ b/windows/bootstrap.sh
@@ -62,7 +62,7 @@ fi
 
 if [ ! -f "bin/OpenAL32.dll" ]; then
     pushd openal-soft-1.16.0/build
-    cmake -G "MSYS Makefiles" -DQT_QMAKE_EXECUTABLE=NOTFOUND -DCMAKE_BUILD_TYPE=Release -DALSOFT_REQUIRE_DSOUND=NO -DCMAKE_INSTALL_PREFIX=$QTOX_DIR/libs ..
+    CFLAGS="-D_TIMESPEC_DEFINED" cmake -G "MSYS Makefiles" -DQT_QMAKE_EXECUTABLE=NOTFOUND -DCMAKE_BUILD_TYPE=Release -DALSOFT_REQUIRE_DSOUND=NO -DCMAKE_INSTALL_PREFIX=$QTOX_DIR/libs ..
     make
     make install
     popd

